src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1954,6 +1962,16 @@ def aks_kollect(cmd,    # pylint: disable=too-many-statements,too-many-locals
         display_diagnostics_report(temp_kubeconfig_path)
 
 
+def _read_periscope_yaml():
+    with open("./deploymentyaml/aks-periscope.yaml", "r") as stream:
```

Comment:
I am not sure if the relative path here will work and my suggestion is to form an absolute path like
```suggestion
    curr_dir = os.path.dirname(os.path.realpath(__file__))
    periscope_yaml = os.path.join(curr_dir, "deploymentyaml", "aks-periscope.yaml")
    with open(periscope_yaml, "r") as stream:
```